FBC/FunctionalEntities/BusinessRegistries.rdf

```diff
@@ -396,7 +390,7 @@
 	<owl:NamedIndividual rdf:about="&fibo-fbc-fct-breg;InactiveStatus">
 		<rdf:type rdf:resource="&fibo-fbc-fct-breg;EntityStatus"/>
 		<rdfs:label>inactive status</rdfs:label>
-		<skos:definition>the entity that was assigned the identifier (business, LEI, or other) is no longer legally registered and/or operating, whether as a result of business closure, acquisition by or merger with another (or new) entity, or determination of illegitimacy</skos:definition>
+		<skos:definition>status indicating that the entity that was assigned the identifier (business, LEI, or other) is no longer legally registered and/or operating, whether as a result of business closure, acquisition by or merger with another (or new) entity, or determination of illegitimacy</skos:definition>
```

Comment:
Should remove "that was assigned the identifier" since the status applies regardless.

Reply:
ok